tests/test_converters/test_obo_graph_to_fhir.py

```diff
@@ -10,28 +13,74 @@
 from tests import IMBO, INPUT_DIR, NUCLEUS, OUTPUT_DIR
 from tests.test_implementations import ComplianceTester
 
-ONT = INPUT_DIR / "go-nucleus.json"
-OUT = OUTPUT_DIR / "go-nucleus.fhir.codesystem.json"
-
 
 class OboGraphToFHIRTest(unittest.TestCase):
-    """Tests OBO JSON -> FHIR."""
+    """Tests OBO JSON -> FHIR. Different ontologies have unique structures, so test some specifics for those."""
+
+    def _load_ontology(self, url: str, download_path: str, use_cache: bool = True) -> GraphDocument:
```

Comment:
Added some stable URLs for reproducible tests.

**Downloads of obographs**
I hesitated on this partially because I don't know if you will like the way this is right now. If we can change OAK a little though so that it is fast to download the source OWL, convert it to Obographs or sqlite, and then run the FHIR converter, that would make this simpler.

**Caching downloads**
Also, I highly prefer the caching as it speeds up development and running of the tests. If there's a different way we want to do this (e.g. pystow), or if you don't prefer caching, we can change. Or if you prefer, I can write these tests within our [TIMS server automation repo](https://github.com/HOT-Ecosystem/tims-ts/pull/1).